samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/ZoomContentControlSamplePage.xaml

```diff
@@ -0,0 +1,60 @@
+﻿<Page x:Class="Uno.Toolkit.Samples.Content.Controls.ZoomContentControlSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Uno.Toolkit.Samples.Content.Controls"
+	  xmlns:utu="using:Uno.Toolkit.UI"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:sample="using:Uno.Toolkit.Samples"
+	  mc:Ignorable="d"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+		<sample:SamplePageLayout x:Name="SamplePageLayout" IsDesignAgnostic="True">
+			<sample:SamplePageLayout.DesignAgnosticTemplate>
+				<DataTemplate>
+					<StackPanel>
+						<utu:ZoomContentControl Grid.Row="1"
+												x:Name="ZoomContent"
+												Width="400"
+												Height="300"
+												HorizontalAlignment="Center"
+												VerticalAlignment="Center"
+												ZoomLevel="1.5"
+												MinZoomLevel="0.5"
+												MaxZoomLevel="3.0"
+												IsZoomAllowed="True"
+												IsPanAllowed="True">
+
+							<StackPanel Orientation="Horizontal"
+										Spacing="12"
+										HorizontalAlignment="Center">
+								<Border BorderThickness="2"
+										BorderBrush="White"
+										Padding="10">
+									<Image Source="ms-appx:///Assets/UnoLogo.png"
+										   Height="101"
+										   Width="75"
+										   HorizontalAlignment="Center" />
+								</Border>
+							</StackPanel>
```

Comment:
the stackpanel serves almost no purpose, considering its removal and put the h-align=center on border